events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -1,40 +1,53 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Trivia</title>
+        <title>Last Avatar Standing</title>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://fonts.googleapis.com/css?family=Squada+One" rel="stylesheet">
         <link href="https://fonts.googleapis.com/css?family=Raleway:300,400,600,700" rel="stylesheet">
         <link rel="stylesheet" href="trivia.css?065">
     </head>
     <body>
-        <h1>Trivia</h1>
-        <input type="button" class="gray" id="begin" value="Let's get started!">
-        <input type="button" class="gray" id="end" value="Game Over">
-        <h3>Questions</h3>
+        <div id="title">
+            <h1 class="size48">LAST<span> AVATAR </span>STANDING</h1>
```

Comment:
why use a `span` in here?

Reply:
This is how I write 'AVATAR' in a different color, it was what I found in looking up CSS techniques, is there something more appropriate to do for that?

Reply:
ah, nice. that's a fine way to do it. it's a little unclear to someone who's looking at the code and only the code that that's what you're doing here. for clarity, i would recommend using an inline style instead of a CSS file selector here (since i'm pretty sure this is the only time you're using that particular CSS selector)

let me know if you have questions about what i mean :)